trillian.proto

```diff
@@ -251,6 +235,15 @@ message SignedLogRoot {
 
   // log_root_signature is the raw signature over log_root.
   bytes log_root_signature = 9;
+
+  reserved 1, 2, 3, 4, 5, 6, 7;
```

Comment:
Could use the `reserved 1 to 7;` syntax?

Reply:
Ah yes, I always forget about that syntax! Done.